news/2024/2024-03-27-osutaiko-world-cup-2024-round-of-32-recap.md

```diff
@@ -0,0 +1,118 @@
+
+Turkey picks up a win with [Namiguru - Zunda Party Night](https://osu.ppy.sh/beatmapsets/1962520#taiko/4067448), but Poland continue to show that they mean business, quickly bringing the score to 1-4. Things are looking bad for Turkey, though Poland also seems to be feeling the pressure, as a narrow loss on [Street - Hypothesis](https://osu.ppy.sh/beatmapsets/2068643#taiko/4533909) gives Turkey a break point and a second wind. The audience rises from their seats, the Twitch chat copies their pastas and Turkey rides that wind, all the way to the first official RO32 Tiebreaker in Taiko World Cup history.
+
+[Yuuni - Frozen Blood](https://osu.ppy.sh/beatmapsets/2152052#taiko/4534309) fittingly sends chills down both teams' spines, and after a well-deserved timeout they begin their final battle. Turkey using 2 mods, Poland only 1. frukoyurdakul seems to have greatly enjoyed the timeout, as despite the added pressure of mods, he brings it home for his team with a 99.86% Hidden Full Combo.
+
+### ::{ flag=DE }:: Germany vs. ::{ flag=AR }:: Argentina
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2100431997&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Looking at Germany's roster for TWC 2024, you'd probably let out a wholeheartedly neutral "huh." Having finished 3rd in 2023, the previous German team decided that this was a good time to let the fresh blood shine with an entirely new roster. A roster that has landed them smack-bang in the middle of the seeding at #15, their Qualifiers results showing a clear preference for reading-based maps such as gimmicks and Hard Rock.
+
+On the other side of the ring, Argentina also presents a completely new roster, putting them only a few steps behind Germany at #18. They share the love for reading maps, but lean more towards the Hidden side of the mod spectrum.
+
+Germany starts off the match with [siqlo - wholeheartedly](https://osu.ppy.sh/beatmapsets/2152039#taiko/4534272), and showcase their Gimmick prowess with a dominant first point. Argentina counters with a whole 6 mods on [Kotoha - Yuki wa Naniiro](https://osu.ppy.sh/beatmapsets/2078626#taiko/4352521) and lets the dust settle into a 1-1 tie. They say lightning never strikes twice, but Germany proceeds to win another 3 maps in a row and bring the score to 4-1. You see where this is going, right? Because Argentina then bounced back. All the way back to, you guessed it, yet another tiebreaker.
```

Comment:
```suggestion
The Germans starts the match off with [siqlo - wholeheartedly](https://osu.ppy.sh/beatmapsets/2152039#taiko/4534272), and showcases their *gimmick* prowess with a dominant first point. Argentina counters with a whole 6 mods on [Kotoha - Yuki wa Naniiro](https://osu.ppy.sh/beatmapsets/2078626#taiko/4352521) and lets the dust settle into a 1–1 tie. They say lightning never strikes twice, but Germany proceeds to win another 3 maps in a row and bring the score to 4–1. You see where this is going, right? Because Argentina then bounced back. All the way back to, you guessed it, yet another tiebreaker.
```